src/simulator/ackermann_model.h

```diff
@@ -0,0 +1,38 @@
+#include "f1tenth_simulator/AckermannCurvatureDriveMsg.h"
+#include <random>
+#include <string>
+
+#include "f1tenth_simulator/AckermannCurvatureDriveMsg.h"
```

Comment:
Remove duplicate headers. Consider [a pass of `clang-format`](https://github.com/ut-amrl/f1tenth_simulator/issues/14#issuecomment-614420142) which will auto sort headers for you.